libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -0,0 +1,538 @@
+    }
+
+    @Override
+    public boolean isUploadingMedia() {
+        return false;
+    }
+
+    @Override
+    public boolean hasFailedMediaUploads() {
+        return false;
+    }
+
+    @Override
+    public void removeAllFailedMediaUploads() {
+    }
+
+    @Override
+    public void removeMedia(String mediaId) {
+    }
+
+    @Override
+    public Spanned getSpannedContent() {
+        return null;
+    }
+
+    @Override
+    public void setTitlePlaceholder(CharSequence placeholderText) {
+    }
+
+    @Override
+    public void setContentPlaceholder(CharSequence placeholderText) {
+    }
+
+    @Override
+    public void onMediaUploadReattached(String localId, float currentProgress) {
+    }
+
+    @Override
+    public void onMediaUploadRetry(String localId, MediaType mediaType) {
+    }
+
+    @Override
+    public void onMediaUploadSucceeded(final String localMediaId, final MediaFile mediaFile) {
+    }
+
+    @Override
+    public void onMediaUploadProgress(final String localMediaId, final float progress) {
+    }
+
+    @Override
+    public void onMediaUploadFailed(final String localMediaId, final MediaType
+            mediaType, final String errorMessage) {
+    }
+
+    @Override
+    public void onGalleryMediaUploadSucceeded(final long galleryId, long remoteMediaId, int remaining) {
+    }
+
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        super.onConfigurationChanged(newConfig);
+        // Toggle action bar auto-hiding for the new orientation
+        if (newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE
+                && !getResources().getBoolean(R.bool.is_large_tablet_landscape)) {
+            mHideActionBarOnSoftKeyboardUp = true;
+            hideActionBarIfNeeded();
+        } else {
+            mHideActionBarOnSoftKeyboardUp = false;
+            showActionBarIfNeeded();
+        }
+    }
+
+    @Override
+    public boolean onTouch(View view, MotionEvent event) {
+        // In landscape mode, if the title or content view has received a touch event, the keyboard will be
+        // displayed and the action bar should hide
+        if (event.getAction() == MotionEvent.ACTION_UP
+                && getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
+            mHideActionBarOnSoftKeyboardUp = true;
+            hideActionBarIfNeeded();
+        }
+        return false;
+    }
+
+    /**
+     * Hide the action bar if needed. Don't hide it if
+     * - a hardware keyboard is connected.
+     * - the soft keyboard is not visible.
+     * - it's not visible.
+     */
+    private void hideActionBarIfNeeded() {
+        ActionBar actionBar = getActionBar();
+        if (actionBar == null) {
+            return;
+        }
+        if (!isHardwareKeyboardPresent()
+                && mHideActionBarOnSoftKeyboardUp
+                && actionBar.isShowing()) {
+            getActionBar().hide();
```

Comment:
We should use the `actionBar` variable instead. Interestingly enough, I believe this can actually throw a `NullPointerException` in some edge cases due to threading :)

```suggestion
            actionBar.hide();
```